docs/plugins.yml

```diff
@@ -0,0 +1,78 @@
+---
+#####################
+# We're excited you wrote plugin! Here's how you share it with others:
+#
+# 1. Add a newline anywhere in the list below
+# 2. List your plugin following the <user>/<repo> format
```

Comment:
@IRCody good call. Our plugin authoring documentation has the naming convention down, but it's worth mentioning here.